widgetry/src/widgets/spinner.rs

```diff
@@ -80,6 +87,9 @@ impl WidgetImpl for Spinner {
             if self.current != self.high {
                 self.current += 1;
             }
+            if self.high < self.current {
```

Comment:
Do we need these new checks in `event`? Since the dropdown only changes by +1 or -1 at a time, the check above should suffice, along with the new fix in the constructor, right?

Reply:
Technically no, what happens if someone manually changes the (pub) current value (not using the modify function), to outside the range?

Reply:
Ah, good point. In that case, we can slightly simplify the previous branch and always do `self.current += 1`

Reply:
Like this?
`            if self.current < self.high {
                self.current += 1;
            } else if self.high < self.current {
                self.current = self.high;
            }`

Reply:
Sort of, I was picturing always doing `+= 1`, then clamping to the max after. I'll adjust in a followup.